src/pop3/POP3Server.ts

```diff
@@ -43,7 +43,8 @@ export default class POP3Server {
 			logger.log(`Received data: ${msg}`)
 			const args = msg.split(" ").slice(1)
 
-			const command = commands.find(c => c.command == msg.substring(0, msg.indexOf(" ")))
+			const sentCmd = msg.includes(" ") ? msg.substring(0, msg.indexOf(" ")) : msg.trim()
+			const command = commands.find(c => c.command == sentCmd)
```

Comment:
I trust you to know what you're doing

Reply:
it means I don't understand what that code is doing and I trust you to have written good code that works xd

Reply:
ok then